cmd/dcrtime_dumpdb/config.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"path/filepath"
+
+	"github.com/jessevdk/go-flags"
+)
+
+const defaultConfigFilename = "dcrtimed.conf"
+
+var (
+	defaultConfigFile = filepath.Join(defaultHomeDir, defaultConfigFilename)
+	defaultBackend    = "filesystem"
+)
+
+// config defines the configuration options for dcrtime_fsck
+//
+// See loadConfig for details on the configuration load process.
+type config struct {
+	HomeDir           string   `short:"A" long:"appdata" description:"Path to application home directory"`
```

Comment:
can we avoid listing all `dcrtimed` params here and use only the subset we need ? 🤔